packages/core-components/src/components/product-list/FilterItem.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+
+export type OptionType = {
+  optionId: string;
+  optionName: string;
+  selected?: boolean;
+};
+
+export type FilterItemType = {
+  placeholder: string;
+  title: string;
+  filterId: string;
+  filterType: string;
+  filterOptions: OptionType[];
+};
+type FilterItemProps = {
+  filterItems: FilterItemType;
+  onClickSort?: (sortParam: string[]) => void;
+};
+
+const FilterItem: React.FC<FilterItemProps> = ({
+  filterItems,
+  onClickSort,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  return filterItems?.filterType && filterItems?.filterOptions ? (
+    <div className='font-primary'>
+      <div className='text-lg/5 font-light'>
+        <p className='font-semibold' style={{ color: '#850000' }}>
+          {filterItems.title}
+        </p>
+        <div className='h-[70px]'>
+          <button
+            id='dropdownButton'
+            data-dropdown-toggle='dropdown'
+            data-dropdown-delay='500'
+            className='w-full p-4 rounded-md border-solid flex justify-end items-center mt-4 hover:bg-gray-200'
+            style={{
+              color: '#850000',
+              borderColor: '#850000',
+              borderWidth: '1px',
+            }}
+            onClick={() => {
+              setIsOpen(!isOpen);
+            }}
+          >
+            <p
+              className='line-clamp-2 font-normal text-sm/4'
+              style={{ color: '#353535' }}
+            >
+              {filterItems.filterOptions.find((item) => item.selected)
+                ?.optionName ?? filterItems.placeholder}
+            </p>
+            <svg
```

Comment:
can we move this one somewhere else

Reply:
can move to config or icon folder but i won't change it so you can move it by your self

Reply:
We can but didn't neccesarily